airbyte-config/config-models/src/main/resources/types/StandardWorkspace.yaml

```diff
@@ -50,3 +50,7 @@ properties:
     type: boolean
   defaultGeography:
     "$ref": Geography.yaml
+  webhookOperationConfigs:
+    description: Configurations for webhooks operations, stored as a JSON object
+    type: object
+    existingJavaType: com.fasterxml.jackson.databind.JsonNode
```

Comment:
should this be pointing to the the WebhookOperationsConfigs class?

Reply:
This has to be a JsonNode to work with the secrets management (because when it's persisted, it won't have the same schema - the secrets will be replaced with coordinates). I'll update the comment to that effect.

Reply:
Does this affect the auto-generated typescript types in the API client? If so, we won't get useful compiler help about breaking changes.

Reply:
Good call-out, but this is actually the internal db representation. The API client gets generated from `airbyte-api/src/main/openapi/config.yaml` which does indeed have some typing. (And indeed I expect will break.)